.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -138,16 +124,22 @@ def buildWindowsManagedImage(String os_series, String img_name_suffix, String la
                                 --image ${AZURE_IMAGE_ID} \
                                 --public-ip-address \"\" \
                                 --nsg-rule NONE
-                        '''
+                            '''
+                        sh """
+                            az vm run-command invoke \
+                                --resource-group ${VM_RG_NAME} \
+                                --name ${VM_NAME} \
+                                --command-id EnableRemotePS
+                        """
                     }
                 }
             }
-
-            stage("Deploy VM") {
+            
+            stage("Setup remote access") {
                 withCredentials([
-                        usernamePassword(credentialsId: JENKINS_USER_CREDS_ID,
-                                            usernameVariable: "SSH_USERNAME",
-                                            passwordVariable: "SSH_PASSWORD")]) {
+                    usernamePassword(credentialsId: JENKINS_USER_CREDS_ID,
+                                        usernameVariable: "SSH_USERNAME",
```

Comment:
nit: should this (and the line below) align with `credentialsId` above?

Reply:
Yes, it definitely should. Thanks for catching this!